### ISSUE TITLE
bug: `<img>` & `<video>` doesn't work with assets in relative paths

### ISSUE THREAD

**Issue by ABausG** [invertase/docs.page#358]
See title.

I was using `<img src="/assets/image.png"/>` which didn't work.

Upon reading the documentation really carefully I noticed that it only mentioned the `<Image>` tag and changing it to that worked.

Is there a particular reason why the Mapping of `<img>` is not working?

Going forward I think there could be two ways:

1. <img> is supported with relative paths
2. A callout is added to the docs hinting that it has to be `<Image>` and not `<img>`. Happy to add that respective callout to the docs if desired :)

**Comment by Ehesp** [comment]
Hey, they should be using the same underlying component, see: https://github.com/invertase/docs.page/blob/main/website/src/components/Markdown.tsx#L43

There's a big update landing tomorrow so I'll double check whether this is still relevant when that lands 👍

**Comment by ABausG** [comment]
Let's say I am planning on doing an initial release of my docs in let's say the next 30 minutes :D Is there something I had to change right away? :D

**Comment by Salakar** [comment]
Seems like this is still an issue at least for video components, a reproduction visible on this page: https://docs.page/donnc/jawce/demo

**Comment by Ehesp** [comment]
Hey @ABausG 

My understanding of MDX is that this is intended behaviour - MDX is treating `video` and `img` as literals which don't get parsed, see the docs: https://mdxjs.com/docs/using-mdx/#components

TLDR:

![image](https://github.com/user-attachments/assets/80934b7b-ca91-44a3-8489-a7abe7f23578)
![image](https://github.com/user-attachments/assets/c0a5b2c9-e54f-4f7a-8cc2-ab4743fa1c1f)

The `<Video />` and `<Image />` components should be used here which pass on the component props of the HTML elements. Is this something you're able to do?

**Comment by ABausG** [comment]
Yes! Was able to use the full <Image> tag.

Notbsure if it might be worth to add a small disclaimer to the docs to indicate that. I had to read the page a couple of times until I noticed it was always talking about Image

**Comment by alexdukeinvertase** [comment]
Hey @ABausG — thanks for your patience on this one, and for the docs disclaimer suggestion.
We're in the middle of shipping docs.page V2 The docs will call out using `/assets/...` from the docs root and the `<Image>` component instead of raw `<img>` with `./` paths. [Docs updates](https://github.com/invertase/docs.page/pull/478/files#diff-b418107e3c83607a6999c963afa21d987096df15a3ae1571fe7aa72c37abe0c9R28-R30) will be pushed soon.
